index.html

```diff
@@ -151,8 +152,9 @@ <h4>Artists and creative types</h4>
                   <h4>Donate to Pursuance!</h4>
                   <p>The Pursuance Project is a nonprofit organization of dedicated volunteers - it needs your financial support. <a class="text-primary" href="donate.html">Every bit helps!</a></p>
                   <h4>Join our Mailing List!</h4>
-                  <p>Our monthly mails include updates on the platform's development and ways to get involved. <a class="text-primary" href="#mc_embed_signup" data-toggle="modal">Sign up here!</a></p>
-                  <button class="subscribe-bottom"><a href="#mc_embed_signup" data-toggle="modal" class="nav-link">Subscribe to our mailing list here!</a></button>
+                  <p>Our monthly mails include updates on the platform's development and ways to get involved. <!--a class="text-primary" href="#mc_embed_signup" data-toggle="modal">Sign up here!</a--></p>
+                  <!--button class="subscribe-bottom"><a href="#mc_embed_signup" data-toggle="modal" class="nav-link">Subscribe to our mailing list here!</a></button-->
+                  <button class="subscribe-bottom btn-lg" data-toggle="modal" data-target="#mc_embed_signup">Subscribe to our mailing list here!</button>
```

Comment:
@howmanywhat Did you mean to comment these out?

Reply:
Yup, feel free to delete what is no longer needed without commenting it out! (I did the same thing before using git, but since git tracks every version of every file, we can "delete" lines with confidence :+1: .)

Reply:
Please join our dev chat server if you haven't already: https://bit.ly/cwc-mminvite

Reply:
Thanks, already did, "anim" and a about a hundred other usernames i tried here were already taken, picking usernames not one of my most favorite things...